src/index.tsx

```diff
@@ -676,6 +703,7 @@ class App extends React.Component<{}, AppState> {
                   draggingElement: null,
                   elementType: "selection"
                 });
+                pushHistoryCurrentEntry();
```

Comment:
The idea is to only store the history on mouseUp. This way we don't store it on every mouse move.